data.json

```diff
@@ -1282,5 +1282,11 @@
     "githubUsername": "Anikcb",
     "imageUrl": "https://user-images.githubusercontent.com/47493387/130086990-1072a097-f8b7-423b-8eeb-b161870b11ea.png",
     "issueNumber": 657
+  },
+  {
+    "name": "Ahamed Basha N",
+    "githubUsername": "ahamedbasha-n",
+    "imageUrl": "https://user-images.githubusercontent.com/69695103/130542739-8b989683-190b-44ee-8c48-cd78b341fbc9.PNG",
```

Comment:
I would suggest more of an image like this. I can't read properly what it says due to the image's size.

![image](https://user-images.githubusercontent.com/51391473/130546663-9ff92a77-14a1-4004-8d9c-dbc67fb027df.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/130546663-9ff92a77-14a1-4004-8d9c-dbc67fb027df.png",
```